ubuntudesign/web-style-guide
# Removed notification component

# Details
- None

## Done
- Removed redundant notifications component file

### QA
- Check scss/patterns/_notifications.scss no longer exists
- Run ```gulp build``` and check there are no errors
